test/paprController/OwnerFunctions.ft.sol

```diff
@@ -42,17 +58,42 @@ contract OwnerFunctionsTest is MainnetForking, UniswapForking {
         IPaprController.CollateralAllowedConfig[] memory args = new IPaprController.CollateralAllowedConfig[](1);
         args[0] = IPaprController.CollateralAllowedConfig(address(nft), true);
 
+        vm.expectEmit(true, false, false, true);
+        emit AllowCollateral(address(nft), true);
         controller.setAllowedCollateral(args);
 
         assertTrue(controller.isAllowed(address(nft)));
     }
 
+    function testSetPoolEmitsCorrectly() public {
```

Comment:
realize i may have gone a bit overboard here

Reply:
yeah this tests seems really long for what's testing?

Reply:
yeah, don't know how else to create a new pool though? method will revert otherwise since invalid pool if we don't create a valid one

Reply:
should be able to just do like this? https://github.com/with-backed/papr/blob/master/test/uniswapOracleFundingRateController/SetPool.ft.sol#L24-L26